inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/InAppMessageViewListenerSpec.kt

```diff
@@ -193,4 +196,88 @@ class InAppMessageViewListenerSpec : BaseTest() {
 
         listener.onTouch(mockView, mockEvent).shouldBeFalse()
     }
+
+    @Test
+        fun `should return true with back button click`() {
+        val mockDisplayManager = Mockito.mock(DisplayManager::class.java)
```

Comment:
It looks like this mock setup could be moved either to a utility function or setup in the `@Before` block to reduce duplication.

Reply:
Moved to class variable